src/frontend/apps/standalone_site/src/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,16 @@
+import { MainLayout } from 'components/Layout';
+import { Header } from 'features/Header';
+import { HomePage } from 'features/HomePage';
+import { Menu } from 'features/Menu';
+import React from 'react';
+
+function AppRoutes() {
+  return (
+    <MainLayout menu={<Menu />}>
```

Comment:
I don't really figure what will become this component once "dynamic". Is it here all the routing will be done ?

Reply:
Yes it is that, some thing like that I guess:

```suggestion
    <MainLayout menu={<Menu />}>
      <Header />
      <BrowserRouter>
        <Switch>
          <Route path={Routes.HomePage} exact>
            <Suspense fallback={<Fallback />}>
              <HomePage />
            </Suspense>
          </Route>
          <Route path={Routes.Favoris} exact>
            <Suspense fallback={<Fallback />}>
              <MesFavoris />
            </Suspense>
          </Route>
          <Route path={Routes.Profil} exact>
            <Suspense fallback={<Fallback />)}>
              <Profil />
            </Suspense>
          </Route>
        </Switch>
      </BrowserRouter>
    </MainLayout>
```

You can here load different `Layout`  depend the route as well, it should stay flexible.